README.md

```diff
@@ -1,11 +1,12 @@
 # NGINX - Helm Charts
 
-This repository contains packaged Helm charts provided by NGINX:
+This repository contains Helm charts to help you install selected NGINX Kubernetes applications. The following are packaged Helm charts provided by NGINX:
 
-- NGINX Ingress Controller
-- NGINX Service Mesh
-- NGINX Appprotect Dos Arbitrator
+NGINX Ingress Controller
+NGINX Service Mesh
+NGINX Appprotect Dos Arbitrator
```

Comment:
Shouldn't this still be a list? Also since you're updating this section there are more charts in the repo now that are not listed.